controllers/storageoscluster/node_operand.go

```diff
@@ -350,6 +350,8 @@ func configureControlPlane(ctx context.Context, stosCl *storageos.Client, cluste
 	}
 	if cluster.Spec.Debug {
 		desiredConfig.LogLevel = storageos.LogLevelDebug
+	} else {
+		desiredConfig.LogLevel = storageos.LogLevelInfo
```

Comment:
nit: I'd avoid the else.  It would be more idiomatic to do:
```
desiredConfig.LogLevel = storageos.LogLevelInfo
if cluster.Spec.Debug {
    desiredConfig.LogLevel = storageos.LogLevelDebug
} 
```

Reply:
Done